fennel/client_tests/branches/test_clone.py

```diff
@@ -171,7 +171,13 @@ def test_clone_errors(client):
     # can not clone to an existing branch
     with pytest.raises(Exception) as e:
         client.clone_branch("main", from_branch="test-branch")
-    assert str(e.value) == "Branch `main` already exists"
+    if client.is_integration_client():
+        assert (
+            str(e.value)
+            == "Server returned: 500, can not clone from branch `random`: does not exist"
+        )
```

Comment:
This error does not look right to me, random branch is not present on line 173 ?

Reply:
changed this.